scripts/zones/Windurst_Waters/npcs/Leepe-Hoppe.lua

```diff
@@ -29,6 +29,18 @@ function onTrigger(player,npc)
 		elseif(MissionStatus >= 1 and MissionStatus < 3) then
 			player:startEvent(0x008e); -- Keep displaying the instructions
 		end
+	elseif(player:getCurrentMission(WINDURST) == AWAKENING_OF_THE_GODS and player:getVar("MissionStatus") == 0) then
+		player:startEvent(0x02DE);
+	elseif(player:getCurrentMission(WINDURST) == AWAKENING_OF_THE_GODS and player:getVar("MissionStatus") == 1) then
+		player:startEvent(0x02DF);
+	elseif(player:getCurrentMission(WINDURST) == AWAKENING_OF_THE_GODS and player:getVar("MissionStatus") == 2) then
+		player:startEvent(0x02E3);
+	elseif(player:getCurrentMission(WINDURST) == AWAKENING_OF_THE_GODS and player:getVar("MissionStatus") == 5 and player:hasKeyItem(BOOK_OF_THE_GODS)) then
+		player:startEvent(0x02E6);
+	else
+		player:startEvent(0x0159); -- Standard Dialogue?
+	end

```

Comment:
Basically everything after that 
```lua
else
    player:startEvent(0x0159); -- Standard Dialogue?
end
```
is going to not exec because 1) the else condition will be met (because it has no conditions) and the entire conditional branch ends at "end" so the next elseif cannot be entered.